packages/wonder-blocks-grid/components/grid.js

```diff
@@ -182,7 +181,7 @@ class GridContext extends React.Component<{
     spec: GridSpec,
 
     // The Row components that will make up the grid
-    children: React.ChildrenArray<Row>,
+    children: React.Node,
```

Comment:
@jeresig I remember you saying something along the lines of the children could be anything.

Reply:
Yep - I changed these in latest modal/grid diff!